### ISSUE TITLE
No SVG utility

### ISSUE THREAD

**Issue by WebReflection** [PolymerLabs/lit-html#65]
There is currently no way to create SVG nodes instead of HTML nodes.

```js
render(html`<rect />`, document.body);
'ownerSVGElement' in document.body.firstElementChild; // false
```

maybe an `svg` tag is needed?

**Comment by kenchris** [comment]
There is already a PR adding support for svg.

**Comment by WebReflection** [comment]
haven't checked PRs, this can be closed once landed then. :+1:

**Comment by justinfagnani** [comment]
Fixed by #57
